docs/nlu/multi-skill-assistants.rst

```diff
@@ -0,0 +1,62 @@
+:desc: Iterate quickly by developing reusable building blocks of AI assistant skills
+       and combining them at training time.
+
+.. _multi_skill_bots:
+
+
+Multi Skill Assistants
+======================
+
+Rasa supports building AI assistants from multiple contextual AI assistant projects.
+This allows it to develop reusable building blocks of skills which you can reuse among
+your projects. One skill could for example handle chitchat, while the other skill
+is responsible for greeting your users. You can develop each skill isolated and then
+import them for the combined training of contextual AI assistant.
+
+An example directory structure could e.g. look like this:
+
+.. code-block:: bash
+
+    .
+    ├── config.yml
+    └── data
```

Comment:
I would think it would make more sense for this dir to be called something like `projects` since each of them already has it's own `data`, and would reflect `building AI assistants from multiple contextual AI assistant projects`

Reply:
or `skills`. I think we should choose that we are either importing skills or projects, and then use that language overall (e.g. below we talk about importing skills, but above talk about building from projects)